ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -3444,6 +3444,18 @@ tests:
     env:
       E2E_RUN_TAGS: '@amd64 and @baremetal-upi and @network-ovnkubernetes and not
         @fips'
+      PATCH_INFRA_MANIFEST: "true"
```

Comment:
Per the comment in https://github.com/openshift/release/pull/42901/files#r1312634851 , you want to set it to "false" or not set it, right?

Reply:
yes, not set it.